x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/metadata/metadata.test.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Metadata, TabProps } from './metadata';
+
+import { useMetadata } from '../../../../metric_detail/hooks/use_metadata';
+import { useSourceContext } from '../../../../../../containers/metrics_source';
+import { createAppMockRenderer } from '../../../../../../test_utils/app_mock_renderer';
+
+jest.mock('../../../../../../containers/metrics_source');
+jest.mock('../../../../metric_detail/hooks/use_metadata');
+
+const metadataProps: TabProps = {
+  currentTimeRange: {
+    from: 1679316685686,
+    to: 1679585836087,
+    interval: '1m',
+  },
+  node: {
+    uuid: 'b9c06da0-b9b0-4c7e-830c-05128cb34396',
+    name: 'host-1',
+    os: 'iOS',
+    title: {
+      name: 'host-1',
+      cloudProvider: 'gcp',
+    },
+    rx: {
+      name: 'rx',
+      value: 0,
+      max: 0,
+      avg: 0,
+    },
+    tx: {
+      name: 'tx',
+      value: 0,
+      max: 0,
+      avg: 0,
+    },
+    memory: {
+      name: 'memory',
+      value: 0.5445920331099282,
+      max: 0.5445920331099282,
+      avg: 0.5445920331099282,
+    },
+    cpu: {
+      name: 'cpu',
+      value: 0.2000718443867342,
+      max: 0.2000718443867342,
+      avg: 0.2000718443867342,
+    },
+    diskLatency: {
+      name: 'diskLatency',
+      value: null,
+      max: 0,
+      avg: 0,
+    },
+    memoryTotal: {
+      name: 'memoryTotal',
+      value: 16777216,
+      max: 16777216,
+      avg: 16777216,
+    },
+  },
+};
+
+describe('Metadata', () => {
```

Comment:
**nit:** To make it simpler to identify what is breaking if this test fails, maybe something like `Hosts Metadata` will give us more context at first sight.

Reply:
Good idea, I set it to `Single Host Metadata (Hosts View)` so it's clear that it's single host metadata inside hosts view. Wdyt?